Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -30,6 +28,7 @@ public ValidationResult GetValidationResult((Contest?, int?, UserInfoModel?, boo
     {
         var (contest, contestId, user, official) = item;
 
+        // TODO: Fix so it uses lecturers in contests business service
         var isUserLecturerInContest = contest != null && user != null && user.IsLecturer;
```

Comment:
This logic is not correct. User can be Lecturer, but not in the specific contest..
Can you log an issue from this comment?